viewer/static/js/app/components/views/dataset-control.js

```diff
@@ -37,10 +37,20 @@ export default function(props) {
       <div className="dataset-cohorts">
         {props.sortedAllPopulations.map(cohort => {
           const isActive = props.sortedPopulationsToShow.includes(cohort);
-          const numClients = props.currentDataset.populations[cohort].total_clients.toLocaleString('en-US');
-          const numPings = props.currentDataset.populations[cohort].total_pings.toLocaleString('en-US');
+          const cohortMeta = props.currentDataset.populations[cohort];
+
+          let numClients = 0;
+          if (cohortMeta.total_clients) {
+            numClients = cohortMeta.total_clients.toLocaleString('en-US');
+          }
+
+          let numPings = 0;
```

Comment:
It seems if the API won't return anything we'll assume 0. Would a string saying "not provided" or something like it be better for the user?

Reply:
Yeah good call